perf/perftool.py

```diff
@@ -35,13 +35,22 @@ def setUp(self):
         Build perftool Test
         Source:
         https://github.com/rfmvh/perftool-testsuite
+        Run perf test
+        Execute perf tests
+        Source : https://github.com/deater/perf_event_tests
+
         '''
 
         # Check for basic utilities
         smm = SoftwareManager()
-        for package in ['gcc', 'make', 'perf']:
+        for package in ['gcc', 'make']:
```

Comment:
sorry my mistake just not i realize that linux-tools-common is required for ubutnu only and perf is required for all distro i think we should have check to handle that , as this won't work